docs/endpoints/multi_user.rst

```diff
@@ -0,0 +1,909 @@
+Multi-User Compute Endpoints
+****************************
+
+Multi-user Compute endpoints (MEP) enable administrators to securely offer compute
+resources to users without mandating the typical shell access (i.e., ssh).  The basic
+thrust is that the administrator of a MEP creates a configuration template, which is
+then populated by user data and passed to a user endpoint process, or UEP.  The UEP runs
+as the local POSIX user, with the only difference from a standard :abbr:`CEP
+([single-user] Compute endpoint)` being that it was started not by a human at the
+command line but on demand by the parent MEP.  In this manner, administrators may preset
+endpoint configuration options (e.g., Torque, PBS, Slurm) and offer user-configurable
+items (e.g., account id, problem-specific number of blocks), while at the same time
+lowering the barrier to use of the resource.
```

Comment:
```suggestion
items (e.g., account id, problem-specific number of blocks), while lowering the barrier to use the resource.
```

Reply:
Wording choice; having slept on it, I prefer the first.